spec/namespaces/cluster.yaml

```diff
@@ -526,13 +526,13 @@ components:
               index:
                 $ref: '../schemas/_common.yaml#/components/schemas/IndexName'
               primary:
-                description: If `true`, returns explanation for the primary shard for the given shard ID.
+                description: When `true`, returns a routing explanation for the primary shard based on node ID.
                 type: boolean
               shard:
                 description: Specifies the ID of the shard that you would like an explanation for.
                 type: integer
                 format: int32
-            description: The index, shard, and primary flag to explain. Empty means 'explain the first unassigned shard'
+            description: The index, shard, and primary flag to explain. Leave this empty to explain the first unassigned shard.
```

Comment:
First sentence: "to explain" => "for which to generate an explanation"? Second sentence: "to explain" => to generate an explanation for"?